docs/theming/theme_scratch.md

```diff
@@ -283,13 +301,17 @@ Let's make use of Bootstrap's layout system and add a `container`, a `row` and s
                     body_class python:plone_layout.bodyClass(template, view);"
         tal:attributes="class body_class;
                         dir python:isRTL and 'rtl' or 'ltr';
-                        python:plone_view.patterns_settings()"
+                        python:context.restrictedTraverse('@@plone_patterns_settings')();"
```

Comment:
i would keep this here in sync with the default main_template and provide plone_view, it's easier to use then.

Reply:
The changes that Stefan did are in sync with the current main_template

Reply:
I updated the template to get the current version.